### PR TITLE
NIFI-12721 Button UX

### PR DESCRIPTION
**Pull request by james-elliott**
Updated all the dialog buttons to change them from stroked and raised to basic. This better aligns with Angular Material guidelines and should address the confusion between disabled and cancelled.

## Before
image

## After
image

# Summary

[NIFI-12720](https://issues.apache.org/jira/browse/NIFI-12720) Button UX

### CODE REVIEW

**Comment by scottyaslan** [comment]
Reviewing...

**Review by scottyaslan — COMMENTED** [review]
@james-elliott thanks for this contribution. These changes are looking good. I found a few other buttons that also need to be updated. The following listing should be all of them:

Screenshot 2024-03-05 at 1 23 02 PM

Here are a few screenshots of some of these buttons as they currently are in this PR:


Screenshot 2024-03-05 at 1 15 51 PM
Screenshot 2024-03-05 at 1 16 31 PM
Screenshot 2024-03-05 at 1 17 24 PM
Screenshot 2024-03-05 at 1 18 30 PM

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss [hunk 1]
```diff
@@ -317,7 +317,7 @@ $appFontPath: '~roboto-fontface/fonts';
         border-color: $accent-palette-A400;
     }
 
-    button:disabled {
+    button.nifi-button:disabled {
```
Comment (scottyaslan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss:320]: Can you please combine this with the style rules directly below this? So then it would be:

```
button.nifi-button:disabled {
        color: $primary-palette-200 !important;
        cursor: not-allowed;
        border: 1px solid $primary-palette-200;

        i {
            color: $primary-palette-200 !important;
        }
    }
```

and there would only be one selector for `button.nifi-button:disabled`.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss [hunk 2]
```diff
@@ -393,7 +393,7 @@ $warn-dark-palette: (
 );
 
 // Define the palettes for your theme
-$material-primary-light: mat.define-palette($material-primary-light-palette);
+$material-primary-light: mat.define-palette($material-primary-light-palette, 600, 100, 900);
```
Comment (scottyaslan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss:396]: What is the purpose of this change?

Comment (james-elliott) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss:396]: The fix is meant to alleviate some of the ambiguity between the disabled action and the cancel actions. This change pushes the primary color here 1 shade darker to increase contrast a bit. You can see the contrast difference in the screens below.

## Before
imageimage

## After
imageimage

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss (hunk 1 above)
Comment (james-elliott) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss:320]: Great catch, I'll update it.
